Makefile

```diff
@@ -25,6 +25,12 @@ define ALL_HELP_INFO
 #   make all
 #   make all HELP=y
 #   make all WHAT=cloudcore
+#   make all WHAT=cloudcore GOLDFLAGS="" GOGCFLAGS="-N -l"
```

Comment:
It would be better to use as env val, like `GOGCFLAGS="-N -l" make all WHAT=cloudcore`.
Besides, empty val should be hide and considered as default.

Reply:
> It would be better to use as env val, like GOGCFLAGS="-N -l" make all WHAT=cloudcore.

makefile will read it as env. so either one is okay to me.

>Besides, empty val should be hide and considered as default.

default config should strip the debug info to keep the binary as small as possible, and this is also align with what k8s does. pls see here, set it as empty string will overwrite the  default string.


https://github.com/kubernetes/kubernetes/blob/14a11060a0775ed609f0810898ebdbe737c59441/build/root/Makefile#L81-L85